text/0000-official-utility-buildpacks.md

```diff
@@ -0,0 +1,75 @@
+# Meta
+[meta]: #meta
+- Name: Officially supported utility buildpacks
+- Start Date: 2021-06-30
+- Author(s): [@jkutner](https://github.com/jkutner)
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+This is a proposal to adopt utility buildpacks as officially supported components of the Cloud Native Buildpacks project.
+
+# Definitions
+[definitions]: #definitions
+
```

Comment:
We have also talked about project descriptor related parts that can be implemented as buildpacks should be owned by cnb as that would ensure that the project only provides implementation for different parts of the spec. I think that would be a good boundary for the kind of utility buildpacks that the project maintains i.e. it should only provide implementations for things that are spec-ed out.